app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    @post = params[:post_id]
+    @user = current_user
+    @like = Like.new(author: @user, post_id: @post)
+
+    @like.save
+    redirect_to user_post_path(@user.id, @post)
+  end
+end
```

Comment:
![image](https://github.com/sumon766/techbright/assets/93039359/ee7def6e-2c31-44b8-a0c7-74d6ca98a0cd)

- [ ] It would be nice if you fix this issue because I have faced it while adding a **like** on a post 😨 Kindly fix it to make your app work as expected 👍

Reply:
@HAMMAS-SALEEM Thanks for the review, in my case, the likes are working perfectly. Please check the screenshot.
![likes](https://github.com/sumon766/techbright/assets/22003499/097ff449-913a-41a7-b504-6059e9c5aca1)

Reply:
Ok Dear, if that's the case you can work on the other changes.

Reply:
I would urge the next reviewer to verify it.

Reply:
Because sometimes some apps won't behave similarly on different systems